lib/_nonstd.scm

```diff
@@ -3067,7 +3067,7 @@
 (define-prim (executable-path)
   (##executable-path))
 
-(define-primitive (##process-mnemonic-identifier)
+(define-primitive (process-mnemonic-identifier)
```

Comment:
For consistency with the other definitions you should do `(define-prim (##process-mnemonic-identifier) ...` rather than `(define-primitive (process-mnemonic-identifier) ...`.

The new style of primitive definition is `(define-primitive ...` but I would prefer having a single refactoring of the `_nonstd.scm` file which switches from `define-prim` to `define-primitive` when the time has come to do that.

Reply:
Ah, okay.  You posted this while I was typing up another (related) comment, so we are a bit off sync.  In a code snippet you shared earlier, you used `(define-primitive (##process-mnenomic-identifier)`.  After I saw that, I figured out the "new style" vs. "old style" distinction, and then assumed you were trying to steer me to the "new style" but had made a typo.